### ISSUE TITLE
Alt-V volume keybinding interferes with readline

### ISSUE THREAD

**Issue by ckuttruff** [termux/termux-app#723]
Firstly, thanks for this amazing project.  I use it all the time.

One quirk I noticed... while Ctl-V (volume down - V) works beautifully but volume up does not page up but loads volume.
While I understand need to access volume, one can easily navigate to diff android app to adjust volume, but having the volume up key more consistent with alt or meta key would I think be most important. 

This is more than just with Alt-V (paging up in emacs / less / etc), but with Alt-D as well  (delete following word)
Here's a cheat-sheet with bindings: http://readline.kablamo.org/emacs.html

I'm happy to try to submit a PR, but since this would change existing behavior, wanted to file an issue.

TL;DR I respectfully suggest that Volume Up key acts more consistently as Alt (or meta key) for better readline support

Thanks again for all ypur work on this excellent terminal emulator!

**Comment by jayesh-bhoot** [comment]
I agree. The main target users of terminux are probably desktop terminal users. So maintaining as much consistency in keybindings as possible wrt terminal/bash/emacs/etc. is a good idea.

I see volume up key as alt/meta too (though the docs probably refrain from christening it as such). Probably many others do too.

A keybinding for volume control without `v` may seem unintuitive, but would also be easily learnable when compared to the inconsistency and unreliability introduced by overriding an established keybinding like `alt-v` or `alt-d` (both heavily used in bash and emacs) when using volume up control.

**Comment by ckuttruff** [comment]
@fornwall, any input here? Happy to submit pr, if folks are open to this change...

**Comment by trygveaa** [comment]
Volume up isn't an alt key in Termux. According to the [key bindings](https://termux.com/touch-keyboard.html), it's used for lots of things, only a few of them is as an alt key.

**Comment by ckuttruff** [comment]
@trygveaa, I'm aware, i read the docs.  This PR suggests a change to improve consistency.  Cause right now how vol up/down are handled is both inconsistent and inconvenient.

Vol down as ctl, vol up as alt / meta makes a ton of sense and would make readline much easier to work with on termux

**Comment by trygveaa** [comment]
How would you replace the other functions that volume up+key has now, if it should only act as an alt key?

**Comment by ckuttruff** [comment]
Some wouldn't have to change (the ones w/o critical readline role).  

Volume can go away in my opinion.  When I'm using vol shortcuts seeing  volume pop up every time I try to page up is confusing.  I think that's an anti-feature that should go away and make clear that user should control volume when a diff app is focused.

To simplify proposal and minimize conflicting keys I would like to distill this:
  * remove vol control shortcut and make VolUp-V be alt-V
  * add any readline shortcuts with alt/meta that don't already have a volume up assignment

Alt-D would be great for delete following word but I see that that would mess with all the arrow key mappings.  I think readline is more critical than arrow keys since they have control character equivalents, but VolUp-v is most important to me

I didn't know about Vol-1, etc for fn keys, and that is certainly useful and not something I'd want to see change
